authentication-typescript/template/src/screens/ExampleHome.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
```

Comment:
Is this page necessary? It's weird that it's separate from the other pages in a different folder...can we just remove it?